services/scan/src/backup.test.ts

```diff
@@ -240,3 +261,18 @@ test('has cvrs.jsonl', async () => {
     `{"1":[],"2":[],"3":[],"4":[],"_ballotId":"abc","_ballotStyleId":"1","_ballotType":"standard","_batchId":"${batchId}","_batchLabel":"Batch 1","_precinctId":"6522","_scannerId":"000","_testBallot":false,"_locales":{"primary":"en-US"},"initiative-65":[],"initiative-65-a":[],"flag-question":["yes"],"runoffs-question":[]}\n`
   );
 });
+
+test('has vx-logs.log', async () => {
```

Comment:
I would add a test where vx-logs.log doesn't exist to test that everything else still gets exported without an error being thrown.

Reply:
Added!